src/Microsoft.Azure.SignalR.Common/Endpoints/AadAccessKey.cs

```diff
@@ -131,41 +129,35 @@ private async Task<bool> HandleHttpResponseAsyncCore(HttpResponseMessage respons
             return true;
         }
 
-        private async Task UpdateAccessKeyAsyncCore(IServerNameProvider provider, ILoggerFactory loggerFactory)
+        private async Task UpdateAccessKeyAsyncCore(IServerNameProvider provider, ILoggerFactory factory)
```

Comment:
>ILoggerFactory factory [](start = 82, length = 22)

move ILoggerFactory to constructor instead of pass as parameter.

Reply:
It's hard for me to do this since the `AadAccessKey` is constructed when parsing `connectionString`.

Reply:
How about move `UpdateAccessKey` method to synchronizor?